endless-mount-dirs.service

```diff
@@ -25,9 +25,10 @@ RequiresMountsFor=/var
 # Only run if the directories don't exist
 ConditionPathIsDirectory=|!/var/endless
 ConditionPathIsDirectory=|!/var/endless-extra
+ConditionPathIsDirectory=|!/var/workdir
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/bin/mkdir -p /var/endless /var/endless-extra
+ExecStart=/bin/mkdir -p /var/endless /var/endless-extra /var/workdir
 ExecStart=-/bin/chown app-manager:app-manager /var/endless /var/endless-extra
```

Comment:
In my first test I checked if the dir would be automatically created, and it is not. Also from Documentation/filesystems/overlayfs.txt

> The "workdir" needs to be an empty directory on the same filesystem
> as upperdir.